[IMP] sales: pdf quote builder revision (#17096)
odoo/documentation: content/applications/sales/sales/sales_quotations/pdf_quote_builder.rst

PDF quote builder
=================

The *PDF Quote Builder* in the Odoo **Sales** app transforms a standard, text-heavy quote into a
polished, branded proposal. Instead of sending a simple table of products and prices, it enables
users to sandwich a quotation between professionally designed PDF pages.

The PDF Quote Builder seamlessly combines any user-provided marketing brochures, technical sheets,

> branded cover pages, and Odoo generated quotes into a single, cohesive document. It also attaches

Reviewer: ```suggestion
branded cover pages, and Odoo-generated quotes into a single document. It also attaches
```

> product-specific files to the quote when a product is added and makes them available on the online

Reviewer: is that an optional setting?

> store.

Reviewer: i'm a bit confused what the quote has to do with the online store? in this case would it be for ecommerce where the quote is the checkout/cart page?

> Quotation templates can also be configured to add specific PDFs to the quote, ensuring the correct

Reviewer: ```suggestion
Quotation templates can also be configured to add specific PDFs to the quote, to ensure
```

> documents reach the intended customers. Dynamic text in PDF templates lets users extract **Sales**

Reviewer: ```suggestion
documents reach the intended customers. Dynamic text in PDF templates allow users to extract **Sales**
```

:guilabel:`Header` to place the PDF before the quote or :guilabel:`Footer` to place the PDF after
the quote.

Further configuration can be done by clicking the :icon:`fa-ellipsis-v` :guilabel:`(vertical
ellipsis)` icon in the top-right corner of the document card and selecting :guilabel:`Edit`.

Click :guilabel:`New` to open a blank document form. In the :guilabel:`File Content` field, click
the :guilabel:`Upload your file` button, and select the PDF to upload. Once uploaded, Odoo
automatically names the document after the file name, and the :guilabel:`Name` field becomes
editable.

To modify the uploaded file, click the :icon:`fa-pencil` :guilabel:`(Edit)`, :icon:`fa-download`
:guilabel:`(Download)`, or :icon:`fa-trash` :guilabel:`(Clear)` icon next to the file name.

If the PDF includes form fields, they are automatically recognized and added as dynamic text fields.
View and edit these fields by clicking :guilabel:`Configure dynamic fields`. For details, refer to
:doc:`pdf_quote_builder/dynamic_text`.

The :doc:`quote_template` field assigns the PDF to the selected template, restricting the PDF's use
to that template.

.. note::

>    Another method to add a header or footer to a quotation template is navigating to

Reviewer: ```suggestion
   Headers and footers can also be added to a quotation template by navigating to
```

   :menuselection:`Sales app --> Configuration --> Quotation Templates`. Select a template and then
   in the :guilabel:`Quote Builder` tab, :guilabel:`Add` or :guilabel:`Upload` a PDF directly.

> .. image:: pdf_quote_builder/config-header-pdf.png

Reviewer: consider `example` admonition or `figcap` to clarify what the reader is looking at
